cmd/orb-cli/followcmd/follow.go

```diff
@@ -189,13 +194,43 @@ func createCmd() *cobra.Command { //nolint:funlen,gocyclo,cyclop,gocognit
 				headers["Authorization"] = "Bearer " + authToken
 			}
 
-			resp, err := common.SendRequest(httpClient, reqBytes, headers, http.MethodPost,
+			result, err := common.SendRequest(httpClient, reqBytes, headers, http.MethodPost,
 				outboxURL)
 			if err != nil {
 				return fmt.Errorf("failed to send http request: %w", err)
 			}
 
-			fmt.Printf("success %s id: %s\n", action, resp)
+			for i := 0; i < maxRetry; i++ {
+				resp, err := common.SendRequest(httpClient, nil, headers, http.MethodGet,
+					fmt.Sprintf("%s/following?page=true", actor))
+				if err != nil {
+					return fmt.Errorf("failed to send http request: %w", err)
+				}
+
+				followingResp := &followingResp{}
+
+				if err := json.Unmarshal(resp, followingResp); err != nil {
+					return err
+				}
+
+				exists := false
+
+				for _, item := range followingResp.Items {
+					if item == to {
+						exists = true
+					}
+				}
+
+				if (action == undoAction && !exists) || (action == followAction && exists) {
+					break
+				}
+
+				if i == maxRetry {
+					return fmt.Errorf("%s failed please check server logs for more info", action)
```

Comment:
how about: "action failed max retries exhausted check server logs for more info"

Reply:
done